### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by vidhanbhatt** [Wang-Siyuan/Yelp-Recommendation-System#9]
This project strives to develop a customized recommendation system of restaurants for users. 

**What I Like**
This is a unique project that I find a good amount of personal value in, so I'm quite curious to know what my list looks like. You have a clear objective and present what you are trying to accomplish in a way that is easy to follow. I like that you have divided the data into a feature matrix and output matrix, and realized that your feature matrix is more sparse than desired. I also like that you are using concepts directly from what we have learned in class, like lasso regularization and logistic regression, to fit your model. In general, I think you adhered to the project specifications well in choosing a cool and important project that is reasonable to solve in one semester, and fits the "prediction" framework of having a training and test set. 

**Room for Improvement**
I'm confused by some of your visualizations. What do the y-axes mean for the second and third histograms? I'm also not sure how useful it is to have the large pie-chart that breaks down each restaurant by cuisine type on Yelp. I think you should reserve your visualizations to better understand your analysis. Instead of just writing your model coefficients down, I would like to see you visualize your fit to the raw data so I can get a sense of how well it performs.

Also, I think you should provide some more justification for why you chose the features you did. Intuitively, they make sense, but for the final project, you should be explicit and say why you think these features are important to include. In the same vein, I would also like to see some justification for why you chose the forms of regression you did by comparing each model's strength with the fundamentals of your dataset. It is clear that you have a technical grasp of what is going on, but I think in a written report it is important to be clear why are you doing what you are doing. Also, just based on the rule of thumb in class, if you have 40,000 reviews in your training set, I'm not sure that you need the same number in your test set.
